frontend/src/scalars/ui/ChartPage.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <div class="visual-dl-chart-page">
+        <!--<ui-expand-panel isShow="{{expand}}" info="{{tagList.length}}" title="{{title}}">-->
+            <div class="visual-dl-chart-page-box">
+                <ui-chart
+                    v-for="tag in filteredTagList"
+                    :tagInfo="tag"
+                    :groupNameReg="config.groupNameReg"
+                    :smoothing="config.smoothing"
+                    :horizontal="config.horizontal"
+                    :sortingMethod="config.sortingMethod"
+                    :downloadLink="config.downloadLink"
+                    :outlier="config.outlier"
+                    :runs="config.runs"
+                    :running="config.running"
+                    :runsItems="runsItems"
+                ></ui-chart>
+            </div>
+            <v-pagination
+                v-if="total > pageSize"
+                v-model="currentPage"
+                :length="total"
+                :total-visible="pageSize"
+            />
+        <!--</ui-expand-panel>-->
+    </div>
+</template>
+<script>
+// import ExpandPanel from '../../common/component/ExpandPanel';
+import Chart from './Chart.vue';
```

Comment:
I think it is possible to import without adding `.vue`

Reply:
thats true